packages/synapse-react-client/src/components/Forum/ForumPage.integration.test.tsx

```diff
@@ -45,35 +24,6 @@ const followRequest: Topic = {
 jest.spyOn(SynapseClient, 'postSubscription')
 jest.spyOn(SynapseClient, 'deleteSubscription')
 
-const forumThread: PaginatedResults<DiscussionThreadBundle>[] = [
```

Comment:
Remove mock data in this test, instead will use existing data in the `mocks` folder. Remove API mocks and replace with `mockServiceWorker`.